app/views/summary/new.html.erb

```diff
@@ -0,0 +1,18 @@
+<h1>
+<small>Summary for:</small><br>
+  <%= @post.title %>
+</h1>
+<div class="row">
+  <div class="col-md-12">
+    <%= form_for [@summary], url: topic_post_summary_index_path(@topic, @post, @summary) do |f| %>
+      <%= f.hidden_field :post_id, value: @post.id %>
```

Comment:
Here's where I send the post id back to controller, enclosed in the `summary:` object.